Userland/Libraries/LibAccelGfx/Context.cpp

```diff
@@ -18,6 +26,36 @@ Context& Context::the()
 
 OwnPtr<Context> Context::create()
 {
+#ifdef AK_OS_MACOS
+    CGLContextObj context = NULL;
+    CGLPixelFormatAttribute attributes[4] = {
```

Comment:
I mean this seems all right, but how are you going to paint without EGL? Did you find a replacement for those APIs in CGL?

Reply:
(PR looks good to me assuming that there's a path forward for that).

Reply:
actually, can we move the details here into helpers? 

```
#ifdef AK_OS_MACOS
  return make_context_macos();
#else 
  return make_context();
#endif
```
big blocks of ifdef code make me sad

Reply:
> I mean this seems all right, but how are you going to paint without EGL?

actually we use EGL only for context creation on linux. we don't need it on macOS because we use CGL there to create OpenGL context.

> actually, can we move the details here into helpers?

sure, I did that.